core/src/main/java/org/mule/runtime/core/internal/processor/chain/SubflowMessageProcessorChainBuilder.java

```diff
@@ -79,21 +81,20 @@ public Location getRootContainerLocation() {
   }
 
   @Override
-  protected MessageProcessorChain createInterceptingChain(Processor head, List<Processor> processors,
-                                                          List<Processor> processorForLifecycle) {
-    return new SubFlowMessageProcessorChain(name, head, processors, processorForLifecycle);
+  protected MessageProcessorChain createSimpleChain(List<Processor> processors,
```

Comment:
I would rename this to `createDefaultChain` or simply `createChain`

Reply:
privileged api...